sdk/eventhub/azure-eventhub/migration_guide.md

```diff
@@ -132,6 +132,8 @@ your program when receiving events.
 In V5, `EventHubConsumerClient` allows you to do the same with the `receive()` method if you
 pass a `CheckpointStore` to the constructor.
 
+> **Note:** V1 checkpoints are not compatible with V5 checkpoints.  If pointed at the same blob, consumption will begin at the first message.  Checkpoint json in the respective blobs can be manually converted (per-partition) if needed.
```

Comment:
I'm wondering if the description here is not accurate, cuz in t1, we use blob lease to maintain the checkpoint, while in t2, we use the metadata of the blob to store the information.